concent_api/middleman/middleman_server.py

```diff
@@ -127,6 +127,8 @@ def _close_middleman(self) -> None:
             await self._respond_to_user(frame, writer)
         except (BrokenEscapingInFrameMiddlemanProtocolError, asyncio.LimitOverrunError, MiddlemanProtocolError) as exception:
             await self._send_immediate_error(writer, exception)
+        except asyncio.streams.IncompleteReadError:
+            pass
```

Comment:
You've added that handling only in `_handle_concent_connection()` (default port:9054), but the exception happen when we close `signing_service` (default port:9055). Either copy or move this to `_handle_service_connection()`.

Reply:
:+1: done

Reply:
Don't we want to add logging for this situation? I'm not sure we should, just asking

Reply:
The case is that as @dybi described it yesterday, the problem is 'by design', so it is not actually an real error. Lets see how the situation with this will develop, maybe after merging his latest changes there will be no need for this pull request.